modules/sched/type.go

```diff
@@ -0,0 +1,77 @@
+//go:build sched
+// +build sched
```

Comment:
@rubiojr - we could consider removing the build tag isolation. There are two mechanisms used for modularity in Risor:

1. Making Risor modules separate Go modules (with their own go.mod)
2. Using Go build tags to avoid building some modules by default

Those two mechanisms can be used independently. The first mechanism should basically always be used. The second mechanism is really there to be a mechanism to avoid the Risor CLI binary size from getting enormous.

When Risor is used as a Go library, just mechanism 1 in isolation means that users can isolate what dependencies they pull in, which is the most important thing.

Given this, would you want to remove the build tag? Both this and the `semver` module don't really add a consequential amount to the CLI build size (unlike, say, the AWS SDK), so mechanism 1 may be enough.

Reply:
@myzie thanks for the great context, I'll remove them, no strong preferences!

Reply:
47c19d66dd82883218f8f15f8a30c9a92671bac9 should change that.

Reply:
Re-reading https://risor.io/docs/contributing_modules, this is clearly articulated there, but I'm guessing this is one of those things where the maintainer may need to weigh in eventually.

Maybe a rule of thumb could be:

> If the new module pulls no new dependecies or a minimal non-cgo dependency chain, our preference would be to include the module in the CLI, without adding build tags

or something along those lines.